src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,196 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import {
+  PersonLink,
+  FatherLink,
+  MotherLink,
+} from '../components/PersonLinks/PersonLinks';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```

Can we use `[]` as default value?
Then you will not need to do `people && people.length > 0` or to use ? in `people?.length`